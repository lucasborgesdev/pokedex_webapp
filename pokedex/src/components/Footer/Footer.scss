@import '../../assets/sass/globals.scss';

.footer {
  background-color: $dark-gray;
  position: relative;
  color: white;
  height: 2rem;
  width: 100%;
  bottom: 0;
  
  .footer-text{
    font-size: 0.8rem;
    position: relative;
    float: right;
    margin-right: 2rem;
    line-height: 2.3em;
    .heart {
      color: $red;
      font-size: 1rem;
    }
  }
}