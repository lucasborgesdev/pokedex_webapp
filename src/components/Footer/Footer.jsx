import React from 'react';
import './Footer.scss';

export default class Footer extends React.Component {
    render () {
        return (
            <div className="footer"> 
                <span className="footer-text">SmarttPokédex made with <span className="heart">♥</span> by Luísa Ribeiro </span>
            </div>
        );
    }
}