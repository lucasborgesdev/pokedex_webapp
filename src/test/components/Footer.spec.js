import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../components/Footer/Footer';

describe('Footer', () => {
  it('Should render correctly', () => {
    const tree = shallow(<Footer />);
    expect(tree).toMatchSnapshot();
  });
});