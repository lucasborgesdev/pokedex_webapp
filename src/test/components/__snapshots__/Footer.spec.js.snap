// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render correctly 1`] = `
<div
  className="footer"
>
  <span
    className="footer-text"
  >
    SmarttPokédex made with 
    <span
      className="heart"
    >
      ♥
    </span>
     by Luísa Ribeiro 
  </span>
</div>
`;
